config-linux.md

```diff
@@ -438,7 +438,7 @@ The following parameters can be specified to setup the controller:
 
 * **`priorities`** *(array, OPTIONAL)* - specifies a list of objects of the priorities assigned to traffic originating from processes in the group and egressing the system on various interfaces.
   The following parameters can be specified per-priority:
-    * **`name`** *(string, REQUIRED)* - interface name
+    * **`name`** *(string, REQUIRED)* - interface name in [runtime network namespace](glossary.md#runtime-namespace)
```

Comment:
> It appears the rest of this file is using footer links…

I think those just slipped through #687 and have filed #799 to catch them up.  @Mashimiao's current inline links match the current style for inline links.